channel/signedstate.go

```diff
@@ -0,0 +1,19 @@
+package channel
+
+import (
+	"github.com/statechannels/go-nitro/channel/state"
+)
+
+type SignedState struct {
+	State state.VariablePart
+	Sigs  map[uint]state.Signature // keyed by participant index
+}
+
+// hasAllSignatures returns true if there are numParticipants distinct signatures on the state and false otherwise.
+func (ss SignedState) hasAllSignatures(numParticipants int) bool {
+	if len(ss.Sigs) == numParticipants {
+		return true
+	} else {
+		return false
+	}
```

Comment:
Similar to above - a container struct like this maybe needs, and almost definitely wants, private data and public setter that vets signatures.

Reply:
Agreed. Because I have work queued up that is based on this branch, I'll punt this for a short while --  #105.